CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## Submitting an Issue
 We use the [GitHub issue tracker](https://github.com/cloudfoundry-community/stackdriver-tools/issues) to track bugs and features.
-Before submitting a bug report or feature request, check to make sure it hasn't already been submitted. You can indicate
+Before submitting a bug report or feature request, check to make sure it has not already been submitted. You can indicate
```

Comment:
nit: our [style guide says contractions are okay](https://developers.google.com/style/contractions), they might even be preferable because it's harder to skip the not part if the word is a contraction as opposed to two separate words.